src/contrib/mlir/compiler.cpp

```diff
@@ -65,17 +69,38 @@
 #include <memory>
 #include <mutex>
 
+// Defines a new LLVM debug type for this file to be used by LLVM_DEBUG macro.
+#define DEBUG_TYPE "mlir-compiler"
+
 using llvm::SmallVector;
 using llvm::StringRef;
 using llvm::make_unique;
 using llvm::ArrayRef;
+
+using namespace ngraph;
 using namespace ngraph::runtime::ngmlir;
 
 static llvm::cl::opt<bool>
-    clEnableAffineLoopFusion("enable-affine-loop-fusion",
+    clEnableAffineLoopFusion("affine-loop-fusion",
                              llvm::cl::init(false),
                              llvm::cl::desc("Enable loop fusion optimization in Affine dialect"));
 
+static llvm::cl::opt<bool>
+    clEnableAffineLoopTiling("affine-loop-tile",
+                             llvm::cl::init(false),
+                             llvm::cl::desc("Enable loop tiling optimization in Affine dialect"));
+
+static llvm::cl::opt<unsigned>
+    clLoopTilingCacheLevel("loop-tile-cache-level",
+                           llvm::cl::init(2),
+                           llvm::cl::desc("Cache level to which to apply loop tiling"));
+
+static llvm::cl::opt<unsigned>
+    clLoopTilingCacheSize("loop-tile-cache-size",
+                          llvm::cl::init(0),
+                          llvm::cl::desc("Cache size to use in loop tiling. Use cache size "
+                                         "provided by TargetTransformInfo by default."));
```

Comment:
Rename options to `affine-loop-tile-*` to be consistent. 
Also, refine that comment to mention that it overrides the cache size inferred from cache-level and TTI

Reply:
> Rename options to `affine-loop-tile-*` to be consistent.

I did this purposely, since we may have different loop tile optimizations (affine, stripe, LLVM, ...) and we could reuse the common configuration flags for all of them. I can change it, though, if you still think it doesn't make sense.

> Also, refine that comment to mention that it overrides the cache size inferred from cache-level and TTI

Sure

Reply:
Better have different knobs for each pass for debugging/performance tuning purposes.